Numsense.UnitTests/DutchExamples.fs

```diff
@@ -0,0 +1,264 @@
+[<InlineData(                                                   "één miljard", 1000000000)>]
+[<InlineData(                                                  "twee miljard", 2000000000)>]
+[<InlineData(  "twee miljard negenenveertig miljoen zeshonderdvijftigduizend", 2049650000)>]
+[<InlineData(
+    "twee miljard honderdzevenenveertig miljoen vierhonderddrieëntachtigduizend zeshonderdzevenenveertig",
+    System.Int32.MaxValue)>]
+let ``tryOfDutch returns correct result`` (dutch, expected) =
+    let actual = Numeral.tryParseDutch dutch
+    Some expected =! actual
+
+[<Theory>]
+[<InlineData(         0, "nul")>]
+[<InlineData(         1, "een")>]
+[<InlineData(         2, "twee")>]
+[<InlineData(         3, "drie")>]
+[<InlineData(         4, "vier")>]
+[<InlineData(         5, "vijf")>]
+[<InlineData(         6, "zes")>]
+[<InlineData(         7, "zeven")>]
+[<InlineData(         8, "acht")>]
+[<InlineData(         9, "negen")>]
+[<InlineData(        10, "tien")>]
+[<InlineData(        11, "elf")>]
+[<InlineData(        12, "twaalf")>]
+[<InlineData(        13, "dertien")>]
+[<InlineData(        14, "veertien")>]
+[<InlineData(        15, "vijftien")>]
+[<InlineData(        16, "zestien")>]
+[<InlineData(        17, "zeventien")>]
+[<InlineData(        18, "achttien")>]
+[<InlineData(        19, "negentien")>]
+[<InlineData(        20, "twintig")>]
+[<InlineData(        21, "eenentwintig")>]
+[<InlineData(        30, "dertig")>]
+[<InlineData(        34, "vierendertig")>]
+[<InlineData(        40, "veertig")>]
+[<InlineData(        42, "tweeënveertig")>]
+[<InlineData(        50, "vijftig")>]
+[<InlineData(        58, "achtenvijftig")>]
+[<InlineData(        60, "zestig")>]
+[<InlineData(        65, "vijfenzestig")>]
+[<InlineData(        70, "zeventig")>]
+[<InlineData(        79, "negenenzeventig")>]
+[<InlineData(        80, "tachtig")>]
+[<InlineData(        86, "zesentachtig")>]
+[<InlineData(        90, "negentig")>]
+[<InlineData(        93, "drieënnegentig")>]
+[<InlineData(       100, "honderd")>]
+[<InlineData(       101, "honderdeen")>]
+[<InlineData(       110, "honderdtien")>]
+[<InlineData(       114, "honderdveertien")>]
+[<InlineData(       135, "honderdvijfendertig")>]
+[<InlineData(       200, "tweehonderd")>]
+[<InlineData(       282, "tweehonderdtweeëntachtig")>]
+[<InlineData(       331, "driehonderdeenendertig")>]
+[<InlineData(       407, "vierhonderdzeven")>]
+[<InlineData(       520, "vijfhonderdtwintig")>]
+[<InlineData(       666, "zeshonderdzesenzestig")>]
+[<InlineData(       798, "zevenhonderdachtennegentig")>]
+[<InlineData(       857, "achthonderdzevenenvijftig")>]
+[<InlineData(       999, "negenhonderdnegenennegentig")>]
+[<InlineData(      1000, "duizend")>]
+[<InlineData(      1001, "duizend een")>]
+[<InlineData(      1010, "duizend tien")>]
+[<InlineData(      1066, "duizend zesenzestig")>]
+[<InlineData(      1337, "duizend driehonderdzevenendertig")>]

```

Comment:
Yes, but not limited to years. You can use for amounts of money, or heights, .... Basically: anywhere where you can write "1337" you can write "dertienhonderdzevenendertig". (btw: Yes, I'm a native speaker)